src/dftbp/dftbplus/main.F90

```diff
@@ -1172,7 +1172,7 @@ subroutine processGeometry(this, env, iGeoStep, iLatGeoStep, tWriteRestart, tSto
       call buildS(env, this%ints%overlap, this%skOverCont, this%coord, this%nNeighbourSk,&
           & this%neighbourList%iNeighbour, this%species, this%iSparseStart, this%orb)
     case(hamiltonianTypes%xtb)
-      @:ASSERT(allocated(this%tblite))
+      @:ASSERT(allocated(this%tblite), Compiled without TBLITE included)
```

Comment:
```suggestion
      @:ASSERT(allocated(this%tblite), "Compiled without TBLITE included")
```
I think, it would be easier to read and more robust, if we quoted the string argument as in Fortran.

Reply:
Done.